File: ServicioTecSND/frmPrincipal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ServicioTecSND
{
    public partial class frmPrincipal : DevExpress.XtraBars.Ribbon.RibbonForm
    {
        public frmPrincipal()
        {
            InitializeComponent();
        }

        private void barButtonItem1_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
        {
            try
            {
                frmOrdenServicio frm = new frmOrdenServicio();
                frm.Show();
            }
            catch (Exception)
            {

                throw;
            }
        }
    }
}

File: ServicioTecSND_vb/frmPrincipal.vb
Public Class frmPrincipal
    Private Sub BarButtonItem1_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles BarButtonItem1.ItemClick
        'Orden de Servicio
        'Dim frm As New frmOrdenServicio()
        'frm.Show()
    End Sub

    Private Sub BarButtonItem2_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles BarButtonItem2.ItemClick
        'Mantenedor de usuarios
        Dim frm As New frmUsuario()
        frm.MdiParent = Me
        frm.Show()
    End Sub
End Class